.github/workflows/lint.yml

```diff
@@ -3,9 +3,9 @@ on:
   push:
     branches: [main]
     paths:
-      - '**.go'
-      - '**.md'
-      - '.github/workflows/lint.yml'
```

Comment:
What does this change do?

Reply:
ehh -- yaml linter in vscode decided to change single to double quotes